src/components/Calculator.js

```diff
@@ -1,43 +1,77 @@
-          <button type="button" className="button-section operator">+</button>
+          <button type="button" className="button-section" value="AC" onClick={() => { this.calculator(digitObj, 'AC'); }}>AC</button>
+          <button type="button" className="button-section" value="+/-" onClick={() => { this.calculator(digitObj, '+/-'); }}>+/-</button>
+          <button type="button" className="button-section" value="%" onClick={() => { this.calculator(digitObj, '%'); }}>%</button>
+          <button type="button" className="button-section operator" value="÷" onClick={() => { this.calculator(digitObj, '÷'); }}>÷</button>
+          <button type="button" className="button-section" value="7" onClick={() => { this.calculator(digitObj, '7'); }}>7</button>
+          <button type="button" className="button-section" value="8" onClick={() => { this.calculator(digitObj, '8'); }}>8</button>
+          <button type="button" className="button-section" value="9" onClick={() => { this.calculator(digitObj, '9'); }}>9</button>
+          <button type="button" className="button-section operator" value="x" onClick={() => { this.calculator(digitObj, 'x'); }}>x</button>
+          <button type="button" className="button-section" value="4" onClick={() => { this.calculator(digitObj, '4'); }}>4</button>
+          <button type="button" className="button-section" value="5" onClick={() => { this.calculator(digitObj, '5'); }}>5</button>
+          <button type="button" className="button-section" value="6" onClick={() => { this.calculator(digitObj, '6'); }}>6</button>
+          <button type="button" className="button-section operator" value="-" onClick={() => { this.calculator(digitObj, '-'); }}>-</button>
+          <button type="button" className="button-section" value="1" onClick={() => { this.calculator(digitObj, '1'); }}>1</button>
+          <button type="button" className="button-section" value="2" onClick={() => { this.calculator(digitObj, '2'); }}>2</button>
+          <button type="button" className="button-section" value="3" onClick={() => { this.calculator(digitObj, '3'); }}>3</button>
+          <button type="button" className="button-section operator" value="+" onClick={() => { this.calculator(digitObj, '+'); }}>+</button>
         </div>
         <span className="end">
-          <button type="button" className="button-section">0</button>
-          <button type="button" className="button-section">.</button>
-          <button type="button" className="button-section operator">=</button>
+          <button type="button" className="button-section" value="0" onClick={() => { this.calculator(digitObj, '0'); }}>0</button>
+          <button type="button" className="button-section" value="." onClick={() => { this.calculator(digitObj, '.'); }}>.</button>
+          <button type="button" className="button-section operator" value="=" onClick={() => { this.calculator(digitObj, '='); }}>=</button>
```

Comment:
- [Optional] Nice job for implementing a click event for all buttons. You can improve on these codes by defining a handleClick event function in which each of your buttons is implemented. You may also decide to define an array of all the button's values, map through each and return the button when clicks on it. This will drastically reduce these bulky codes and enhance performance.